autosubmit/autosubmit.py

```diff
@@ -1385,25 +1387,17 @@ def expid(description, hpc="", copy_id='', dummy=False, minimal_configuration=Fa
         # Create the experiment structure
         Log.info("Generating folder structure...")
 
-        exp_folder = os.path.join(root_folder, exp_id)
+        exp_folder = root_folder / Path(exp_id)
         try:
-            os.mkdir(exp_folder)
-            os.mkdir(os.path.join(exp_folder, "conf"))
-            os.mkdir(os.path.join(exp_folder, "pkl"))
-            os.mkdir(os.path.join(exp_folder, "tmp"))
-            os.mkdir(os.path.join(exp_folder, "tmp", "ASLOGS"))
-            os.mkdir(os.path.join(exp_folder, "tmp", "LOG_"+exp_id))
-            os.mkdir(os.path.join(exp_folder, "plot"))
-            os.mkdir(os.path.join(exp_folder, "status"))
-            # Setting permissions
-            os.chmod(exp_folder, 0o755)
-            os.chmod(os.path.join(exp_folder, "conf"), 0o755)
-            os.chmod(os.path.join(exp_folder, "pkl"), 0o755)
-            os.chmod(os.path.join(exp_folder, "tmp"), 0o755)
-            os.chmod(os.path.join(exp_folder, "tmp", "ASLOGS"), 0o755)
-            os.chmod(os.path.join(exp_folder, "tmp", "LOG_"+exp_id), 0o755)
-            os.chmod(os.path.join(exp_folder, "plot"), 0o755)
-            os.chmod(os.path.join(exp_folder, "status"), 0o755)
+            # Setting folders and permissions
+            exp_folder.mkdir(mode = 0o755)
```

Comment:
PyCharm is warning about the style issue here, @VindeeR . Actually, in every line here, as normally parameters don't have spaces wrapping the `=`. It's a bit hard to see, but there are squiggly lines surrounding the equal signs.

![image](https://github.com/user-attachments/assets/a9dea290-8d0e-460b-951c-fdc0268ca070)

Are you seeing the same in your PyCharm? If not we may have different settings, and it'd be good o align (something that follows pep/linter default rules).